core/build.sbt

```diff
@@ -124,7 +122,10 @@ lazy val akkastreamTestkit =
     .settings(
       (sourceDirectory in AvroConfig) := baseDirectory.value / "src/test/avro",
       (stringType in AvroConfig) := "String",
-      javacOptions += "-Xlint:unchecked"
+      javacOptions += "-Xlint:unchecked",
+      javaOptions += "-XX:MaxMetaspaceSize=512M",
```

Comment:
Please put these changes in a separate PR, they are not related to reading from all partitions.

Reply:
Did this only to be able to run tests locally, removing